### ISSUE TITLE
Include project path in free-text dataset search results

### ISSUE THREAD

**Issue by travissarles** [SwissDataScienceCenter/renku-graph#724]
As a renku-graph API user, I'd like to be able to have a direct link to a dataset image from all the free-text dataset search, the dataset details and the project datasets resources.

**The problem:**
Currently, API users cannot easily find the direct link to the dataset image defined in its metadata. In the easiest case, they have to know the link to the project repo in GitLab and the way how to get to a file inside a repo. In the worst-case scenario, they have to also jump through few APIs to find a project where the dataset exists.

**Acceptance criteria:**
* the free-text dataset search, the dataset details and the project datasets resources to return a link to the image like follows:
  ```json
  ...
      "images": [
        {
          "location": "image.png",
          "_links":[  
             {  
                "rel":  "view",
                "href": "https://renkulab.io/gitlab/project_path/raw/master/data/mniouUnmal/image.png"
             }
          ]
        }
      ],
  ...
  ```

**Comment by jachro** [comment]
Done but merged into the `660-new-provisioning-process` so it will be available once this branch is merged to master.
